css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
```

Comment:
Why `rectangle`? What type of `rectangle` is this class referring to? Could we find a name that's better suited?